pkgs/development/libraries/gstreamer/default.nix

```diff
@@ -47,7 +47,7 @@
 
   gst-plugins-ugly = callPackage ./ugly { inherit CoreFoundation DiskArbitration IOKit; };
 
-  gst-plugins-viperfx = callPackage ./viperfx { };
+  gst-plugins-viperfx = throw "'gst_all_1.gst-plugins-viperfx' was removed as it is broken and not maintained upstream"; # Added 2024-12-16
```

Comment:
Should respect `config.allowAliases`， we can refer to gnome's approach:

https://github.com/NixOS/nixpkgs/blob/948a31b4b765c01f485c05bdeb12b2197f440d5a/pkgs/desktops/gnome/default.nix#L24-L26

Reply:
hmm. I updated it but thinking perhaps should just remove the alias all together as it makes it impossible to build the attributes set. eg:

```
$ nix-build -A gst_all_1
error: 'gst_all_1.gst-plugins-viperfx' was removed as it is broken and not maintained upstream
```

which seems somewhat undesirable.